NAMESPACE

```diff
@@ -278,8 +278,10 @@ importFrom(grid,viewport)
 importFrom(gridExtra,arrangeGrob)
 importFrom(lifecycle,badge)
 importFrom(methods,.valueClassTest)
+importFrom(methods,`slot<-`)
```

Comment:
we can also just import the whole `methods` package. otherwise it gets complicated